docs/source/get_involved/code_of_conduct.rst

```diff
@@ -28,40 +29,65 @@ conform to the following Code of Conduct.
+of technical content and open community governance. All community members are expected
+to adhere to the principle of free speech in academia and science, and, to the best
+of their abilities, contribute to jointly creating
+the best possible science and the best possible open community.
+
+We are committed to creating a friendly and respectful place for learning,
+teaching and contributing. All participants in our in-person events and online
+communications are expected to show respect and courtesy to others at all times.
 
 To make clear what is expected, everyone participating in activities
 associated with the sktime project is required to conform to this Code
 of Conduct. This Code of Conduct applies to all spaces managed by the
 sktime project including, but not limited to, in-person focus groups and
 workshops, and communications online via GitHub.
 
-The lead investigator of the sktime - Dr Franz Király - is responsible
-for enforcing the Code of Conduct. He can be contacted by emailing
-f.kiraly@ucl.ac.uk.
+The Code of Conduct Committee of sktime is responsible for defining and
+enforcing the Code of Conduct, according to Section 4 below.
+
+Incidents can be reported to the Code of Conduct Committee as outlined in Section 3, and
+are handled in accordance with Sections 3 and 4.
+
+The community council of sktime (see current composition on "roles")
+is responsible for ensuring that resolutions of the Code of Conduct committee
+are adhered to, and can be contacted under info@sktime.org.
 
-Reports may be reviewed by other members of the core development team,
-unless there is a conflict of interest, and will be kept confidential.
 
 2 Code of Conduct
 -----------------
 
 The sktime team are dedicated to providing a welcoming and supportive
 environment for all people, regardless of background or identity. As
-such, we do not tolerate behaviour that is disrespectful to our
-community members or that excludes, intimidates, or causes discomfort to
-others. We do not tolerate discrimination or harassment based on
-characteristics that include, but are not limited to: gender identity
-and expression, sexual orientation, disability, physical appearance,
-body size, citizenship, nationality, ethnic or social origin, pregnancy,
-familial status, veteran status, genetic information, religion or belief
-(or lack thereof), membership of a national minority, property, age,
-education, socio-economic status, technical choices, and experience
-level.
+such, we do not tolerate:
+
+* behaviour that is disrespectful to our
+  community members or that excludes, intimidates, or causes discomfort to others.
+* discrimination or harassment based on protected characteristics, including
+  restriction of fair opportunities based on protected characteristics
+* silencing of reasonable opinions and stifling of free speech on topics within
+  the project scope (science and open community operations)
```

Comment:
I dont get what this part could be concretely, so I would either make it more clear or remove it. Like this it can be anything? What is "silencing", just expressing another opinion might already then be "silencing" another opinion?

Reply:
> What is "silencing", just expressing another opinion might already then be "silencing" another opinion?

No, that is not the case as per intention.

I can see how the formulation is vague, i.e., what is "silencing", what is "reasonable". I removed that.